rmw_connext_shared_cpp/src/node.cpp

```diff
@@ -78,7 +73,8 @@ create_node(
   // since the participant name is not part of the DDS spec
   // the node name is also set in the user_data
   size_t length = strlen(name) + strlen("name=;") +
-    strlen(namespace_) + strlen("namespace=;") + 1;
+    strlen(namespace_) + strlen("namespace=;") +
+    strlen(context->options.security_context) + strlen("securitycontext=;") + 1;
```

Comment:
@ivanpauno I saw this being called `contextname` elsewhere. Any reason not to standardize?

Reply:
It was supposed to be `securitycontext` everywhere. I forgot to update some places.

Reply:
See https://github.com/ros2/rmw_connext/pull/392/commits/3d53fafa80a7da972c4d6ed48e04ffdac45cc866.